Resources/templates/CommonAdmin/ExcelAction/ExcelBuilderAction.php.twig

```diff
@@ -60,11 +71,21 @@ class ExcelController extends \{{ namespace_prefix }}\{{ bundle_name }}\Controll
             200, array()
         );    
         $response->headers->set('Content-Type', 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet; charset=utf-8');
-        $response->headers->set('Content-Disposition', 'attachment;filename={{ builder.filename }}.xlsx');
+        $response->headers->set('Content-Disposition', 'attachment;filename='.$fileName);
```

Comment:
This is a BC-breaking change, which causes the default export to be broken (there is no extension anymore). The static `.xlsx` is also not good (this probably means that using other types than Excell2007 was already broken). 

I suggest to add auto-guessing behavior based on the `fileType` parameter (so, Excel2007 -> .xlsx, Excel5 -> .xls and more). Maybe there is already something like that available in the ExcelBundle. You should also check the header, because that specific header is also only for Office2007 files.